src/algorithms/bubbleSort/BubbleSort.java

```diff
@@ -0,0 +1,22 @@
+package src.algorithms.bubbleSort;
+
+public class BubbleSort {
+    /**
+     * Sorts the given array.
+     * @param arr array to be sorted.
+     * @return the same array arr that is sorted.
+     */
+    public static int[] sort(int[] arr) {
+        int n = arr.length;
+        for (int i = 0; i < n - 1; i++ ) {
```

Comment:
This is optional but really helpful in subsequent more complex tasks. can add short in-line comment with # to explain this is the outer loop that ...